rust/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.55-slim-buster
+FROM rust:1.61-slim-buster
```

Comment:
there's a benchmark dockerfile that probably needs to be updated too: https://github.com/swift-nav/libsbp/blob/master/python/Dockerfile.benchmark

Reply:
hmmmmm that shouldnt affect the test error `./rust/bin/sbp2json: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.25' not found (required by ./rust/bin/sbp2json)` but maybe updating that to python3.10? or make it install `apt get musl` as well will have to test that

Reply:
This is using GLIBC (not MUSL) -- this error happens when the glibc the tool is linked against is newer than the glic that the system suports -- try changing `FROM python:3.7-slim-stretch` to `FROM python:3.7-slim-buster`.

Reply:
@adrian-kong Note that the released version of `sbp2json` (the one that goes to the releases page) does use musl, but this is so that we can release a statically linked version of the tool that doesn't require a specific version of glibc.

Reply:
> @adrian-kong Note that the released version of `sbp2json` (the one that goes to the releases page) does use musl, but this is so that we can release a statically linked version of the tool that doesn't require a specific version of glibc.

Actually, this bit about using musl isn't true... I thought it was, but apparently not:
```
✦ ❯ ldd ~/.local/share/swift-cli/packages/libsbp/4.1.6/contents/sbp2json
        linux-vdso.so.1 (0x00007fff1cdce000)
        libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f1f6d491000)
        librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007f1f6d487000)
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f1f6d464000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f1f6d272000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f1f6d893000)
```